docs/public/specifications/tests/DNSSEC-TP/dnssec18.md

```diff
@@ -216,13 +216,14 @@ None.
 [DS18_NO_MATCH_CDS_RRSIG_DS]:            #summary
 [Default level]:                         ../SeverityLevelDefinitions.md
 [ERROR]:                                 ../SeverityLevelDefinitions.md#error
+[Get-Parent-NS-IP]:                      ../MethodsV2.md#method-get-parent-ns-ip-addresses
```

Comment:
Ok. Could you provide an appropriate fix?

Reply:
No, since there is no such method in [MethodsV1]. I suggest that it is kept as-is and broken. Or the link is removed and a comment is added.

[MethodsV1]: https://github.com/zonemaster/zonemaster/blob/master/docs/public/specifications/tests/Methods.md